From: Ruben Undheim <ruben.undheim@gmail.com>
Date: Tue, 25 Oct 2016 08:35:06 +0200
Subject: StrictHostKeyChecking=no

---
 python/gitrocd | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/gitrocd b/python/gitrocd
index bff1d14..a8f507f 100755
--- a/python/gitrocd
+++ b/python/gitrocd
@@ -163,7 +163,7 @@ def clone_repo(url, urlstr, reponame, version=1):
 umask %s
 mkdir -p %s/%s
 cd %s/%s
-git clone --bare %s%s%s %s%s
+GIT_SSH_COMMAND="ssh -o StrictHostKeyChecking=no" git clone --bare %s%s%s %s%s
 cd %s%s
 git config --local remote.origin.fetch "+refs/heads/*:refs/heads/*"
 """ % (umask, cachearea, urlstr, cachearea, urlstr, url, reponame, suffix, reponame, suffix, reponame, suffix)
