From: Ruben Undheim <ruben.undheim@gmail.com>
Date: Sat, 5 Nov 2016 09:51:43 +0100
Subject: Does not need mode == 1 to record in db

---
 python/gitrocd | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/gitrocd b/python/gitrocd
index bed83d0..bf17cbf 100755
--- a/python/gitrocd
+++ b/python/gitrocd
@@ -407,7 +407,7 @@ def handle_one(number, action, version, url, reponame, branch, clientqueue, mode
             istag = False
         rowrevparse = None
         notfoundindb = False
-        if istag and mode == 1:
+        if istag:
             dblock.acquire()
             check_if_db_open()
             cursor.execute(
